### PR TITLE
Add redux, Add extensions

### PR DESCRIPTION
**Pull request by Flaque**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,25 +44,27 @@
     "fuse.js": "^3.1.0",
     "immutable-props": "^1.1.0",
     "isdev": "^1.0.1",
-    "knex": "^0.13.0",
+    "knex": "^0.14.0",
```
Comment (kmccrohan) [inline package.json]: is this version change necessary? It might not be compatible with other dep versions we have

File: package.json [hunk 2]
```diff
@@ -36,25 +44,27 @@
...
     "linklocal": "^2.8.1",
     "lodash": "^4.17.4",
     "prop-types": "^15.5.10",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
     "source-map-support": "^0.5.0",
-    "sqlite3": "^3.1.13",
+    "sqlite3": "https://github.com/mapbox/node-sqlite3/tarball/master",
```
Comment (kmccrohan) [inline package.json]: why are we doing this? have you verified this works on another computer by deleting their node_modules and reinstalling? If you want I can try it

Comment (QuantamHD) [reply package.json]: This is pulling off nightly bad idea. We should be using node-pre-gyp leave it as is.

**Comment by Flaque** [comment]
Fixed those issues, all still works. Check on your comps, or just merge it in <3

**Pull request merged**